validator/src/cli.rs

```diff
@@ -468,7 +468,8 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .value_name("NUMBER")
                 .takes_value(true)
                 .default_value(&default_args.full_snapshot_archive_interval_slots)
-                .help("Number of slots between generating full snapshots")
+                .help("Number of slots between generating full snapshots. \
+                    Must be a multiple of the incremental snapshot interval.")
```

Comment:
What do you think of:
```
Must be a multiple of --incremental-snapshot-interval-slots value
```
I mention this as I see `--no-incremental-snapshots` specifically mention args like this, but I'm fine with what you have as well. I don't think we have a hard rule on this

Reply:
Yeah true, I used the more explicit value but I see your point.